### ISSUE TITLE
Same messages exist on all profiles regardless of whose account it is

### ISSUE THREAD

**Issue by kaxline** [Methodician/guytime#75]

**Comment by kaxline** [comment]
@bsteinbeigle Can you add the screenshots to this issue that you put in the Google Doc? Also, can you share the Google Doc with `keith@yachtcopter.com`?

**Comment by bsteinbeigle** [comment]
@kaxline screenshots uploaded. This was a brand new profile that I just created. That said, shouldn't have any messages populate provided none were sent. Let me know if this is helpful. Thank you! 

![Fellas Message Issue 1](https://user-images.githubusercontent.com/89544670/131711677-996bf1c1-c46a-4428-b290-5be93540a502.jpg)
![Fellas Message Issue 2](https://user-images.githubusercontent.com/89544670/131711689-bef0c297-7570-4a8e-a1ca-40159dd71d49.jpg)

**Comment by bsteinbeigle** [comment]
Google doc. link was shared. May receive the link twice by accident. Let me know if received.

**Comment by kaxline** [comment]
@bsteinbeigle This one is tricky. It seems that when you see those chats, your user is indeed a member of that group chat. So it's a matter of people being added to group chats when they shouldn't be.

I think we should keep an eye on this as we go about fixing other stuff and the culprit will become more clear.

We should probably implement a way for users to leave a group chat. That way we can at least clean this up when it happens.

**Comment by bsteinbeigle** [comment]
@kaxline agreed if you believe it will be more efficient to hold off until further down the road. In the interim, agreed with developing a way to remove yourself from a chat.
